mathics/builtin/atomic/atomic.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Atomic Primitives
+"""
+
+from mathics.builtin.base import (
+    Builtin,
+    Test,
+)
+
+
+class AtomQ(Test):
+    """
+    <dl>
+    <dt>'AtomQ[$expr$]'
+        <dd>returns 'True' if $expr$ is an expression which cannot be divided into subexpressions, or 'False' otherwise.
+
+        An expression that cannot be divided into subparts is called called an "atom".
+    </dl>
+
+    Numbers are atoms:
+    >> AtomQ[1.2]
+     = True
+
+    So are symbolic constants:
+    >> AtomQ[2 + I]
+     = True
+
+    A 'Symbol' not bound to a value is an atom too:
+    >> AtomQ[x]
+     = True
+
+    On the other hand, expressions aren't atoms:
+    >> AtomQ[2 + Pi]
```

Comment:
The same as my comment in the documentation PR.

Reply:
Should be fixed now the same way it was done in the documentation PR. But please double check.

Reply:
2 + Pi is clearly not an atom. Consider the following example:

```
a= 2+Pi;
Unprotect[Pi]; Pi=3;
a
```
Then, ``a`` evaluates to 5. 

Again, the main property of an atomic expression is that if you evaluate it, you get always the same expression, independently of what are the state of the ``Definitions`` object.

Reply:
From https://reference.wolfram.com/language/ref/AtomQ.html:

> An expression which cannot be divided into subexpression [after evaluation]

I'll make this more explicit that there is more than one part after evaluation.